docs/debugging.md

```diff
@@ -73,22 +77,33 @@ You may use commands such as `crane`, `oras`, `crictl` or `docker` to verify the
 2023/03/17 04:45:49 --> GET https://registry.k8s.io/v2/pause/blobs/sha256:61fec91190a0bab34406027bbec43d562218df6e80d22d4735029756f23c7007 [body redacted: omitting binary blobs from logs]
 2023/03/17 04:45:49 --> GET https://prod-registry-k8s-io-us-west-2.s3.dualstack.us-west-2.amazonaws.com/containers/images/sha256%3A61fec91190a0bab34406027bbec43d562218df6e80d22d4735029756f23c7007 [body redacted: omitting binary blobs from logs]
 ```
+
 From my location, the pull command accesses registry.k8s.io, us-west1-docker.pkg.dev and prod-registry-k8s-io-us-west-2.s3.dualstack.us-west-2.amazonaws.com. You will need to have DNS and HTTP access to these domains on your node to pull images.
 
+It's also possible to run these commands on your node if you don't have SSH access by using `kubectl run`:
+
+```sh
+kubectl run --rm -it crane --restart=Never --image=gcr.io/go-containerregistry/crane --overrides='{"spec": {"hostNetwork":true}}' -- pull --verbose registry.k8s.io/pause:3.9 pause.tgz
```

Comment:
I'd say:
```suggestion
kubectl run --rm -it crane --restart=Never --image=gcr.io/go-containerregistry/crane --overrides='{"spec": {"hostNetwork":true}}' -- pull --verbose registry.k8s.io/pause:3.9 /dev/null
```

Except I guess windows nodes ...

Reply:
Does the crane image have windows? If not I'd say /dev/null

Reply:
I don't think so, this only shows different Linux archs.  https://console.cloud.google.com/artifacts/docker/go-containerregistry/us/gcr.io/crane/sha256:3065f35363454047665af6be10c3aa49717097f8cb3bf061df91eeb9ac92eb62;tab=manifest


I'm not a Windows person either though :)